alerting/src/utils/server-utils.ts

```diff
@@ -0,0 +1,186 @@
+import fetch from 'node-fetch';
+import moment from 'moment';
+import { xml2js } from 'xml-js';
+import { get, isEmpty, isString, union } from 'lodash';
+import { Extent, WCSRequestUrl } from './raster-utils';
+import { AlertConfig } from '../entities/alerts.entity';
+
+// eslint-disable-next-line fp/no-mutation
+process.env.NODE_TLS_REJECT_UNAUTHORIZED = '0';
+
+const xml2jsOptions = {
+  compact: true,
+  trim: true,
+  ignoreComment: true,
+};
+
+export function formatUrl(
+  baseUrl: string,
+  params: { [key: string]: any } = {},
+): string {
+  const url = new URL(baseUrl);
+  Object.keys(params).forEach((k) => url.searchParams.append(k, params[k]));
+  return url.toString();
+}
+
+type AvailableDates = {
+  [key: string]: number[];
+};
+
+/**
+ * Format the raw data to { [layerId]: availableDates }
+ * @param rawLayers Layers data return by the server 'GetCapabilities' request
+ * @param layerIdPath path to layer's id
+ * @param datesPath path to layer's available dates
+ * @returns an object shape like { [layerId]: availableDates }
+ */
+function formatCapabilitiesInfo(
+  rawLayers: any,
```

Comment:
Any way to avoid the `any` use here? Will it be solved once we have types from `frontend`?